packages/client/src/Loc.ts

```diff
@@ -1184,27 +1206,60 @@ export class AcceptedRequest extends ReviewedRequest {
         return await this.refresh() as OpenLoc
     }
 
+    private checkOpenParams(): OpenPolkadotLocParams {
+        const requesterAddress = this.request.requesterAddress;
+        if (requesterAddress === undefined || requesterAddress?.type !== "Polkadot") {
+            throw Error("Only Polkadot requester can open LOC");
+        }
+        return {
+            locId: this.locId,
+            legalOfficer: this.owner,
+        }
+    }
+
+    async estimateFeesOpen(): Promise<FeesClass> {
+        if (this.request.locType === "Transaction") {
+            return this.locSharedState.client.estimateFeesOpenTransactionLoc(this.checkOpenParams())
+        } else if (this.request.locType === "Identity") {
+            return this.locSharedState.client.estimateFeesOpenIdentityLoc(this.checkOpenParams())
+        } else {
+            throw Error("Collection LOCs fees are estimated with estimateFeesOpenCollection()");
```

Comment:
Maybe this is equivalent but the `new` keyword is missing.

Reply:
Indeed. That being said, it is also mentioned to always use `throw new ...` as `new` is required with ES6 classes (just a good habit I guess, and maybe that at some point TypeScript will also drop support for the case without `new`).